app/api/gradeSimulation/historyGraduationId/view/func.js

```diff
@@ -0,0 +1,23 @@
+const _ = require('lodash')
+const app = require('@/app')
+
+module.exports = async function (context) {
+  const GradeSimulation = app.model.gradeSimulations;
+  const { historyGraduationId } = context.params
+  const season = app.helpers.season.findSeasonKey()
+
+  app.helpers.validate.throwMissingParameter('historyGraduationId', context.params)
+
+  return await GradeSimulation
+    .find({
+      active: true,
+      historyGraduation: String(historyGraduationId),
+      $or: [
+        { year: { "$gt": season.year } },
+        {
+          year: season.year,
+          quad: { "$gt": season.quad }
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555384)
```suggestion
          quad: { '$gt': season.quad }
```